src/main/java/org/jboss/arquillian/extension/jacoco/client/filter/AndFilter.java

```diff
@@ -16,13 +16,18 @@
     @Override
     public boolean include(T object)
     {
+        boolean include = false;
         for (Filter<T> f : filters)
         {
             if (!f.include(object))
             {
+                if (f.toString().contains("IncludeRegExpPaths") && include) {
```

Comment:
If there are multiple include patterns, we should include the asset matching any of them. 
The problem was that if the asset does not match other include patterns, it returns false, thus excluding the asset.

The check ensures that, if there are multiple include patterns and if a asset matches either of the include filter pattern, it should be included.